app/models/post.rb

```diff
@@ -0,0 +1,16 @@
+class Post < ApplicationRecord
+  after_save :update_posts_counter
+  belongs_to :user
```

Comment:
- [ ] it would be better if you change `user` to `author`. as in the diagram in order to match the requirements you can do that like the below example: 
```ruby
`belongs_to :author, class_name: 'User'`
```

Reply:
I will make the changes required although, I was approved on previous PR. I would say it's kind of misleading to name it as so, since we are dealing with users of a blog. But learning something new 😄